lib/util/git/set-branch-commit.ts

```diff
@@ -0,0 +1,30 @@
+import { getCache } from '../cache/repository';
+import type { BranchCache } from '../cache/repository/types';
+import { getBranchCommit } from '.';
+
+export function setBranchCommit(
```

Comment:
Document that this function is called after a new commit is made to a branch (either creating a branch or updating it)